test/test_unit.jl

```diff
@@ -586,6 +586,198 @@ isdir(outdir) && rm(outdir, recursive=true)
     @test_throws ArgumentError TimeSeriesCallback(semi, [1.0 1.0 1.0; 2.0 2.0 2.0])
   end
 
+  @timed_testset "Consistency check for HLL flux (naive)" begin
```

Comment:
Could you maybe wrap each equation part inside a corresponding `@testset "Equation name" begin ... end` block?

Reply:
Addressed in 8dff75c549be5a97b1db6209eaae3a276cde956a.